shell/components/ResourceTable.vue

```diff
@@ -150,6 +150,14 @@ export default {
     }
   },
 
+  mounted() {
+    window.addEventListener('keyup', this.handleEnterKeyPress);
```

Comment:
Please add a note as reference to this issue/PR, where the plugin used for the shortcuts does not allow propagation to other components (e.g. Namespace) and we are forced to use the basic JS API.